Day 1 - Any two numbers from the list add up to K

```diff
@@ -0,0 +1,39 @@
+Question :
+
+Given a list of numbers and a number k, return whether any two numbers from the list add up to k.
+For example, given [10, 15, 3, 7] and k of 17, return true since 10 + 7 is 17.
+
+public class addition_two_number {
+
+    static boolean Logic(String[] split, Long value) {
```

Comment:
```suggestion
    static boolean searchString(String[] split, Long value) {
```

1. Always follow standard definition for method names
2. Better to provide visible name to method names, so that if you have multiple approaches in future you can just add meaningful function names.